Sources/NIO/System.swift

```diff
@@ -445,3 +445,23 @@ internal enum KQueue {
     }
 }
 #endif
+
+#if os(Linux)
```

Comment:
@toffaletti can you just move this to `Linux.swift` ? This already has linux specific code.